platform/functionality/resolve/resolve.go

```diff
@@ -23,16 +24,28 @@ func HandleResolve(pattern string, sr schema.Registry, ir elasticsearch.IndexRes
 		sourcesToShow = &sourcesFromElasticsearch
 	}
 
+	var filteredOut []elasticsearch.Index
+	for _, index := range sourcesToShow.Indices {
+		if index.Name == common_table.VirtualTableElasticIndexName {
```

Comment:
```suggestion
 		// don't include the common table in the results - it's internal table used by Quesma, and should be exposed as an index / data stream
		if index.Name != common_table.VirtualTableElasticIndexName {
		        filtered = append(filtered, index)
 		}
 		
```

Reply:
I think this is simpler to read rather than `continue`, but may be just my preference :)